app/models/placements/school.rb

```diff
@@ -81,4 +81,8 @@ class Placements::School < School
   has_many :partner_providers,
            through: :partnerships,
            source: :provider
+
+  def primary?
+    phase == "Primary"
+  end
```

Comment:
I think this would be better off in the parent `School` class rather than the namespaced `Placements::School`. We already have [`primary_or_secondary_only?`](https://github.com/DFE-Digital/itt-mentor-services/blob/9102ee3579eec8dd7a2184e646a51d76f4f38518/app/models/school.rb#L120-L122) defined in the parent, and this `primary?` method feels suitably similar.

We should also use the constant `PRIMARY_PHASE`:

```suggestion
  def primary?
    phase == PRIMARY_PHASE
  end
```